packages/web/src/components/gcds-page-feedback/gcds-page-feedback.tsx

```diff
@@ -0,0 +1,649 @@
+  #sanitizeOptions = {
+	CUSTOM_ELEMENT_HANDLING: {
+	  tagNameCheck: /^gcds-/, // only allow GCDS custom elements
+	  attributeNameCheck: /.+/, // allow all attributes
+	  allowCustomizedBuiltInElements: false // customized built-ins are not allowed yet
+	}
+  };
+
+  @State() submissionURL: string = "";
+
+  @Prop() action?: string;
+  validateActionUrl() {
+	this.submissionURL = this.action;
+  }
+  @Prop({ mutable: true }) feedbackForm?: '';
+  @Watch('feedbackForm')
+  validateFeedbackForm() {
+
+	// Check if we need to load an external feedback form
+    if ( this.feedbackForm ) {
+	
+	  // Check if there is not user configured form, if so we abort the external fetch
+	  const slotFeedbackFormInjected = this.el.querySelector( 'div[slot=feedback-form]' );
+	  if ( slotFeedbackFormInjected && !slotFeedbackFormInjected.hasAttribute( 'data-ajaxed-from' ) ) {
+		console.error( 'A custom feedback form are already set, unable to override via ajax' );
+		return;
+	  }
+	  
+	  // Fetch the file
+	  fetch( this.feedbackForm, {
+		method: 'get'
+	  }).then( ( response ) => {
+	  
+		// If we get a 404, let's abort and use default feedback form
+		if (!response.ok) {
+		  this.feedbackForm = '';
+		  throw new Error( "404 - file not found" );
+		}
+		
+		// Let's process the received response body
+		return response.text();
+	  })
+	  .then( ( responseText ) => {
+
+		// Create the slot container and parsing the response text
+		const divSlotCustomForm = document.createElement('div');
+		divSlotCustomForm.slot = 'feedback-form';
+		divSlotCustomForm.dataset.ajaxedFrom = this.feedbackForm;
+		divSlotCustomForm.innerHTML = DOMPurify.sanitize( responseText, this.#sanitizeOptions ); // Insert sanitized HTML only
+
+		// Extract the custom action from the fetched HTML fragment
+		const elmWithActionURL = divSlotCustomForm.querySelector( '[data-feedback-form-action]' ) as HTMLElement;
+
+		// Remove the submission URL not manually overridden 
+		if ( !this.action ) {
+		  this.submissionURL = "";
+		}
+
+		// Set the submission URL if not specified
+		if ( !this.submissionURL && elmWithActionURL ) {
+		  this.submissionURL = elmWithActionURL.dataset.feedbackFormAction;
+		}
+		
+		// Abort if there no custom form action url at this step
+		if ( !this.submissionURL ) {
+		  console.error( 'A custom feedback form action URL is required' );
+		  return;
+		}
+
+		// Remove, if existing, the div slot with the custom form
+		slotFeedbackFormInjected && slotFeedbackFormInjected.remove();
+		
+		// Add the new div slot to this element
+		this.el.append( divSlotCustomForm );
+		
+		// Reset and keep a clean copy of the custom form
+		if ( this.#feedbackFormFragment.childElementCount > 0 ) {
+		  this.#feedbackFormFragment = new DocumentFragment();
+		}
+		this.#feedbackFormFragment.append( divSlotCustomForm.cloneNode( true ) );
+	  });
+	}
+  }
+
+  
+  /*
+   * In-page operation to gather some metadata required by the page feedback tool
+   */
+  #pageTitle = '';
```

Comment:
Love the comments! Could we remove all the `#` at the start of variables and function names? It's not a common pattern in this codebase and it would be great if we can keep the consistency

Reply:
>  Could we remove all the # at the start of variables and function names?

Yes I will replace them by the declaration `private pageTitle`. 

FYI - Those variable prefixed with `#` was private properties as per how it is defined in the JS language. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties